hezar/preprocessors/preprocessor.py

```diff
@@ -82,10 +77,7 @@ def load(
                     config_file = os.path.join(hub_or_local_path, subfolder, f)
                 else:
                     config_file = hf_hub_download(
-                        hub_or_local_path,
```

Comment:
@HamedBabaei  How does this not conform to the wrapping rules?

Reply:
@arxyzan nice catch, I checked out this and it seems to be black was causing the issue, so I fixed it manually (and tested to make sure it will not change it) for this and the rest of the catch that you provided, and it works now!

Reply:
@HamedBabaei  Thanks for your efforts <3